windows/win_robocopy.ps1

```diff
@@ -0,0 +1,86 @@
+#!powershell
+# This file is part of Ansible
+#
+# Copyright 2015, Corwin Brown <blakfeld@gmail.com>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# WANT_JSON
+# POWERSHELL_COMMON
+
+$params = Parse-Args $args;
+
+$result = New-Object psobject @{
+    win_robocopy = New-Object psobject @{
+        recurse         = $false
+        purge           = $false
+    }
+    changed = $false
+}
+
+$robocopy_opts = @()
+If (Get-Member -InputObject $params -Name src) {
```

Comment:
Please use Get-Attr / Get-AnsibleParam as per guidelines - it will make the code smaller.